code/headers/lidar_packets.hpp

```diff
@@ -1,45 +0,0 @@
-#pragma once
```

Comment:
Why is this file deleted?

Reply:
it shouldn't be in my feature branch

Reply:
I returned the file and corrected the outdated usart.